app/assets/stylesheets/was_seed.scss

```diff
@@ -18,9 +17,7 @@
 .#{$namespace}-was-thumb-item {
   cursor: pointer;
   float: left;
-  height: 100%;
-  margin-bottom: 10px;
-  width: 220px;
+  margin: 2px 2px 10px 2px; //top right bottom left
```

Comment:
Shorthand form for property `margin` should be written more concisely as `2px 2px 10px` instead of `2px 2px 10px 2px`